Makefile

```diff
@@ -261,11 +261,15 @@ update-book-cli: ## Update book cli documentation.
 
 .PHONY: maxperf
 maxperf: ## Builds `reth` with the most aggressive optimisations.
-	RUSTFLAGS="-C target-cpu=native" cargo build --profile maxperf --features jemalloc,asm-keccak
+	RUSTFLAGS="-C target-cpu=native -Z threads=8" cargo +nightly build --profile maxperf --features jemalloc,asm-keccak
```

Comment:
```suggestion
	RUSTFLAGS="-C target-cpu=native -Z threads=$(nproc)" cargo +nightly build --profile maxperf --features jemalloc,asm-keccak
```

Reply:
oo, cool

Reply:
but if you don't have the command installed it's not so cool. besides, don't think anyone with less cores builds maxperf tbh.

Reply:
iirc it's a unix command, so everyone should have it. you can additionally do ` || 8`

Reply:
not available on macos

Reply:
> not available on macos

insane